scripts/build/config.mjs

```diff
@@ -387,6 +398,7 @@ const universalFiles = [...nonPluginUniversalFiles, ...pluginFiles].flatMap(
       platform: "universal",
       buildOptions,
       isPlugin,
+      withDts,
```

Comment:
Aren't we copy every file twice?

Reply:
Oh thank you. I've addressed your review [a829a3b](https://github.com/prettier/prettier/pull/14212/commits/a829a3b56c16a872784f6c706dc1457af2de0583)